src/applications/combined-debt-portal/debt-letters/components/PaymentHistoryTable.jsx

```diff
@@ -89,4 +93,8 @@ const PaymentHistoryTable = ({ currentDebt }) => {
   );
 };
 
+PaymentHistoryTable.propTypes = {
+  currentDebt: PropTypes.object.isRequired,
```

Comment:
Can we add the other props we use from `currentDebt` like `benefitType` and `originalAr` just to make sure we've got everything coming in as expected!

Reply:
This cleared all of the lint errors but sure.